cms/djangoapps/contentstore/views/tests/test_block.py

```diff
@@ -852,17 +852,17 @@ def test_duplicate_event(self):
         XBLOCK_DUPLICATED.connect(event_receiver)
         usage_key = self._duplicate_and_verify(self.vert_usage_key, self.seq_usage_key)
         event_receiver.assert_called()
-        self.assertDictContainsSubset(
-            {
-                "signal": XBLOCK_DUPLICATED,
-                "sender": None,
-                "xblock_info": DuplicatedXBlockData(
-                    usage_key=usage_key,
-                    block_type=usage_key.block_type,
-                    source_usage_key=self.vert_usage_key,
-                ),
-            },
-            event_receiver.call_args.kwargs,
+        duplicated_event = {
+            "signal": XBLOCK_DUPLICATED,
+            "sender": None,
+            "xblock_info": DuplicatedXBlockData(
+                usage_key=usage_key,
+                block_type=usage_key.block_type,
+                source_usage_key=self.vert_usage_key,
+            ),
+        }
+        self.assertTrue(
+            duplicated_event.items() <= event_receiver.call_args.kwargs.items()
         )
```

Comment:
I believe you can do:

```python
assert  duplicated_event.items() <= event_receiver.call_args.kwargs.items()
```

Or

```python
self.assertLessEqual(duplicated_event.items(), event_receiver.call_args.kwargs.items())
```

But trying use `assertTrue` here will cause pylint to flag it.

Reply:
This is perfect. I'll make those changes. Thank you.